org.eclipse.lsp4e/src/org/eclipse/lsp4e/LanguageServiceAccessor.java

```diff
@@ -144,13 +144,20 @@ public boolean isActive() {
 
 	public static @NonNull List<CompletableFuture<LanguageServer>> getInitializedLanguageServers(@NonNull IFile file,
 			@Nullable Predicate<ServerCapabilities> request) throws IOException {
+			return getInitializedLanguageServers(file, request, true);
+	}
+
+	public static @NonNull List<CompletableFuture<LanguageServer>> getInitializedLanguageServers(@NonNull IFile file,
+			@Nullable Predicate<ServerCapabilities> request, boolean connectFile) throws IOException {
```

Comment:
I suggest `forceConnectFile` instead, as the file may already be connected. Also, do you have any suggestion what could be a better term than `connect` ?

Reply:
Thanks for the review. I have done the changes.

You mean, a better name for `LanguageServerWrapper#connect`? I guess `open` would be a better name, in the end the effect of it is to created the document synchronizer which sends a didOpen notification, isn't it?. I do not know if it is worth renaming the methods now that they already exist.